t/t-fsck.sh

```diff
@@ -177,3 +182,28 @@ begin_test "fsck detects invalid pointers"
   grep 'pointer: unexpectedGitObject: "large.dat".*should have been a pointer but was not' test.log
 )
 end_test
+
+begin_test "fsck operates on specified refs"
+(
+  set -e
+
+  reponame="fsck-refs"
+  setup_invalid_pointers
+
+  git rm -f crlf.dat large.dat
+  git commit -m 'third commit'
+
+  git commit --allow-empty -m 'fourth commit'
+
+  git log --format=oneline
+
+  # Should succeed.  (HEAD and index).
+  git lfs fsck
+  git lfs fsck HEAD
+  git lfs fsck HEAD^^ && exit 1 || true
+  git lfs fsck HEAD^
+  git lfs fsck HEAD^..HEAD
+  git lfs fsck HEAD^^^..HEAD && exit 1 || true
+  git lfs fsck HEAD^^^..HEAD^ && exit 1 || true
```

Comment:
As elsewhere in this PR, I think these should work as just `&& exit 1`.

Reply:
Fixed.